src/config.ts

```diff
@@ -60,4 +60,5 @@ export const config: BotConfig = {
         type: 'WATCHING',
         value: 'for commands.',
     },
+    pageStyleHelpCommand: true
```

Comment:
I don't know if this is necessary but it's fine for now.

Reply:
It was the cleanest way that I saw fit for doing this implementation

Reply:
I meant I don't know if it needs to be a toggle.

Reply:
I wanted to keep the new and old functionality, as some people prefer a pagination based help menu while others prefer the long embed version

Reply:
And if not a config toggle, what would it be then?